nntrainer/include/tensor.h

```diff
@@ -110,13 +111,28 @@ class Tensor {
   float getValue(unsigned int batch, unsigned int c, unsigned int h,
                  unsigned int w);
 
+  /**
+   * @brief     Scalar multiplication and apply immediatly
+   * @param[in] value Scalar value to be multiplied
+   * @retval    #ML_ERROR_NONE  Successful
+   * @retval    #ML_ERROR_INVALID_PARAMETER Tensor dimension is not right
+   */
+  int multiply_i(float const &value);
+
```

Comment:
I think this is not related with this PR. You would be better to apply it other PR.

Reply:
Thanks. It is updated :)